pkg/resources/kafka/configmap.go

```diff
@@ -250,3 +255,116 @@ func (r Reconciler) generateBrokerConfig(id int32, brokerConfig *v1beta1.BrokerC
 
 	return strings.Join(completeConfig, "\n")
 }
+
+func (r *Reconciler) postConfigMapReconcile(log logr.Logger, desired *corev1.ConfigMap) error {
```

Comment:
This method updates broker status so updateBrokerStatus might be a more descriptive name

Reply:
Renamed